pom.xml

```diff
@@ -464,6 +464,13 @@
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-logging-logback</artifactId>
         <version>0.123.0-alpha</version>
+        <scope>import</scope>
```

Comment:
This artifact is a `jar`. So, you don't need import scope.

Reply:
Got it